### ISSUE TITLE
Adjustments to gray background and secondary text color for accessibility

### ISSUE THREAD

**Issue by mceledonia** [patternfly/patternfly#2424]
# Problem
### Part 1
Currently our secondary text color is not accessible over our standard grey background. It is close, and we can make slight adjustments to nudge it into passing territory without introducing a breaking change. This impacts various components which benefit from the text-color hierarchy of both primary and secondary when directly on the gray background such as forms and empty state.

### Part 2
We currently have 3 light grey background variables with no reason to have more than 1. In an effort to reduce confusion and improve consistency across our portfolio, narrow this down to a single grey background variable. This will depend on the outcome of part 1. This should be investigated to ensure we don't introduce a breaking change by removing the excess background color variables.

### The numbers
- Background colors
`$pf-global--BackgroundColor--150: $pf-color-black-150: #f5f5f5`
`$pf-global--BackgroundColor--200: $pf-color-black-100: #fafafa `
`$pf-global--BackgroundColor--300: $pf-color-black-200: #ededed `

- Secondary global color
`$pf-global--Color--200: $pf-color-black-600: #737679`

- Contrast ratios, target `4.5`
`#f5f5f5` on `#737679` is `4.19` = Fail
`#fafafa` on `#737679` is `4.38` = Fail
`#ededed` on `#737679` is `3.9` = Fail

# Solution
### Background color
**Our new background color would be `#F0F0F0`**

In order to achieve that: 
1. Change `$pf-global--BackgroundColor--200` to call `$pf-color-black-200`
2. Change `$pf-color-black-200` to `#F0F0F0`
3. Remove `$pf-global--BackgroundColor--150` and `$pf-global--BackgroundColor--300`

Note: This change would remove `#EDEDED` entirely from the system, which is totally fine. It would not affect any other gray variables, and by replacing ED with F0 we keep the color cadence of our grayscale in tact. I chose to replace ED because it was the color which was closest to the new F0.

### Secondary color
**Our new secondary color would be `#6A6E73`**

In order to achieve that: 
1. Change `$pf-color-black-600` to call `#6A6E73`

Note: Changing our secondary color seems like a major move but it looks generally the same in all instances. See before/afters below.

### All together
`#6A6E73` on `#F0F0F0` makes a perfect `4.5` Pass

### Visual before/after
The empty state is a good example of where this change comes in handy / has the most impact. Hint: It looks the same! Which is a great thing.

<img width="681" alt="image" src="https://user-images.githubusercontent.com/30935938/68237276-d26cb500-ffd4-11e9-8e50-dbd9b2c22551.png">
<img width="638" alt="image" src="https://user-images.githubusercontent.com/30935938/68237283-d7c9ff80-ffd4-11e9-9c75-79786eac0349.png">

## FYI
@mcoker @matthewcarleton @gdoyle1 @mmenestr @jknilans @rachael-phillips @andybraren @jgiardino @LHinson

**Comment by mceledonia** [comment]
Updated with solution proposal. @mcoker @rachael-phillips @LHinson

**Comment by mceledonia** [comment]
@rachael-phillips @mcoker Do we know when we can make these changes? If we can slide in the non-breaking changes I think we will be in a better spot for updating the sketch library as well.

**Comment by mcoker** [comment]
@mceledonia @rachael-phillips I think we can go ahead and update the black color vars since the changes are so slight. This introduces the risk of users that may be using the existing hex values for those colors in their application, and those colors would no longer match our usage of the colors in Patternfly. But that is a common risk we have when making updates like this in general, and have made this sort of update in the past. Users should be using the CSS color variables we have for these colors (or the SCSS var if they can't find an appropriate CSS var) - not the hex values directly.

**Comment by mcoker** [comment]
@mceledonia @rachael-phillips FYI we chose not to update the global black colors before the holiday break and will look at this again after the new year and after mentioning it in a community meeting, or possibly in our next breaking change release.

**Comment by mcoker** [comment]
closed via https://github.com/patternfly/patternfly/pull/2818
